tardis/io/schemas/montecarlo_definitions.yml

```diff
@@ -5,7 +5,6 @@ definitions:
       title: 'Damped Convergence Strategy'
       type: object
       additionalProperties: false
-      properties:
```

Comment:
Is this a relevant bugfix to this PR?

Reply:
Not exactly. But while working on this PR I saw this small error and fixed it. Do you think I should put this in a separate commit?

Reply:
Yes. We prefer to limit PRs to fix one item at a time whenever possible.